fleetshard/pkg/testutils/k8s.go

```diff
@@ -72,14 +72,17 @@ func (t reconcileTracker) Create(gvr schema.GroupVersionResource, obj runtime.Ob
 		route.Status = admittedStatus()
 	}
 	if err := t.ObjectTracker.Create(gvr, obj, ns); err != nil {
-		return err
+		return fmt.Errorf("adding GVR %q to reconcile tracker: %w", gvr, err)
 	}
 	if gvr == centralsGVR {
 		var multiErr *multierror.Error
 		multiErr = multierror.Append(multiErr, t.ObjectTracker.Create(secretsGVR, newCentralTLSSecret(ns), ns))
 		multiErr = multierror.Append(multiErr, t.ObjectTracker.Create(routesGVR, newCentralRoute(ns), ns))
 		multiErr = multierror.Append(multiErr, t.ObjectTracker.Create(routesGVR, newCentralMtlsRoute(ns), ns))
```

Comment:
```suggestion
		multiErr = multierror.Append(multiErr, errors.Wrap(t.ObjectTracker.Create(secretsGVR, newCentralTLSSecret(ns), ns), "creating central TLS secret"))
		multiErr = multierror.Append(multiErr, errors.Wrap(t.ObjectTracker.Create(routesGVR, newCentralRoute(ns), ns), "creating central route"))
		multiErr = multierror.Append(multiErr, errors.Wrap(t.ObjectTracker.Create(routesGVR, newCentralMtlsRoute(ns), ns), "creating central MTLS route"))
```

could have something like this, WDYT?

Reply:
this suggestion needs importing `"github.com/pkg/errors"` btw

Reply:
Thanks for looking into this! But, I just realized that this is code for the test suite, which does not seem to be used in the production code, hence I would say that we shouldn't worry about the wrapping here. Let's keep this simple.